classes/tables/entries_table.php

```diff
@@ -51,12 +64,18 @@ public function col_name($record) {
                 '/mod/cpdlogbook/delete.php',
                 ['cmid' => $this->cmid, 'id' => $record->id, 'sesskey' => sesskey()]
         );
+
+        $updatestr = get_string('update', 'mod_cpdlogbook');
+        $deletestr = get_string('delete', 'mod_cpdlogbook');
+
+        // The pix_icons use default moodle icons.
+        $menu = new action_menu();
```

Comment:
I'd split this out into a new column called 'Actions' (also use this string from moodle core) on the far right